apps/codeforafrica/src/lib/data/utils/posts.js

```diff
@@ -117,29 +118,54 @@ export async function getPost(api, slug, primaryTag) {
     image: coverImage,
     ...meta,
   };
-  return {
-    title,
-    blocks: [
+  const blocks = [
+    {
+      authors: authors.map(({ fullName, bio }) => {
+        return {
+          name: fullName,
+          bio,
+        };
+      }),
+      title,
+      coverImage,
+      excerpt,
+      tags,
+      publishedOn: formatDate(publishedOn, {
+        includeTime: false,
+        month: "short",
+      }),
+      primaryTag,
+      blockType: "article",
+      ...other,
+    },
+  ];
+
+  const globalPostSettings = await api.findGlobal("settings-publication");
+  const primaryTagGlobalSettings = globalPostSettings?.[primaryTag] ?? {};
+  const { showRecent, title: recentTitle } = primaryTagGlobalSettings;
```

Comment:
```suggestion
  const publicationSettings = await api.findGlobal("settings-publication");
  const primaryTagPostSettings = publicationSettings?.[primaryTag] ?? {};
  const { showRecent, title: recentTitle } = primaryTagPostSettings;
```

No need to use global.